.github/workflows/release.yml

```diff
@@ -1,7 +1,8 @@
 name: Release Node.js Package
 on:
   release:
-    types: [created]
+    types: [published]
```

Comment:
If we create release note as draft and publish it, current workflow doesn't run. We should use `published`. 
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release